src/components/Modal/index.tsx

```diff
@@ -125,6 +125,9 @@ class Modal extends React.Component<ThemedModalProps, any> {
 				responsive={false}
 				margin="small"
 				animate={false}
+				style={{
+					marginTop: theme.header.height,
+				}}
```

Comment:
I think you can also use the prop margin without style and get the same result: 
``` margin={{
    "horizontal": "small",
    "top": theme.header.height,
    "bottom": "small",
  }}```

Reply:
You can only use the breakpoints you define using Grommet's API (e.g. re-assign [these values](https://github.com/grommet/grommet/blob/1ee98897864208d6b3ec0462cc9bb5f8c994484a/src/js/utils/index.d.ts#L41)) in the `margin` property

Reply:
From the file you shared I can see that it should actually accept both 
`margin?: MarginType;`
`export type MarginType = EdgeType;`
´´´
type EdgeType = "none" | EdgeSizeType | {bottom?: EdgeSizeType | string,end?: EdgeSizeType | string,horizontal?: EdgeSizeType | string,left?: EdgeSizeType | string,right?: EdgeSizeType | string,start?: EdgeSizeType | string,top?: EdgeSizeType | string,vertical?: EdgeSizeType | string} | string
```
where `top: EdgeSizeType | string`should accept `theme.header.height` which should be a string

Reply:
It should, seeing [this issue](https://github.com/grommet/grommet/issues/2456), but it doesn't for some reason

Reply:
or rather the value is accepted but totally ignored

Reply:
It is overwritten by the default value of the `position` prop, which is `center` and puts the modal in an absolute position with `top: 50%`, a value that is overwriting the setted margin: {top: ...}.
The `margin` prop in this case is just changing the `top` `bottom` `left` `right` value of the modal position and not the css margin value. To make it work you should set `position="top"` as a Layer prop or otherwise remove the `margin` prop since in this case it is not doing much.